pegasus/sites.v3/hourofcode.com/styles/030-font-awesome-min.css

```diff
@@ -1,4 +1,6 @@
+@import url("https://dsco.code.org/assets/font-awesome-pro/1682441295/css/fontawesome.min.css");
```

Comment:
Is there a way to avoid hardcoding this url in so many places?

Reply:
@cat5inthecradle I went back through in [this commit]([d71d918](https://github.com/code-dot-org/code-dot-org/pull/51490/commits/d71d918b87d00e8cfd0b88e944e85d55529b120e)) and tried to reuse file paths where possible. I had to turn various CSS files into SCSS files to do this -- one exception is hourofcode.com, where we use this funky route to combine css when we request /style.css in our code:

https://github.com/code-dot-org/code-dot-org/blob/cfa114efdb6cd0d41ebeba9895e7226b93d4752a/pegasus/router.rb#L240

I updated docs to reference which places we'll need to change file paths (I'll update the dashboard PR to use the shared constants as well).

Take a quick look to make sure I didn't do anything egregious with this little last minute refactor 😬 -- I did go back through code.org, hourofcode.com, and advocacy.code.org and saw DSCO URLs loading when I loaded each page.

Reply:
hmm, i see. The change looks valid. I forgot for a bit that there isn't a super smooth way to do that without switching to scss or implementing CSS variables for the first time in the repo. Your call whether it's cleaner. I was trying think ahead to whoever ends up changing the url - since they'll still probably need to search the repo to find all the spots, maybe keeping it lighter css is better. Sorry for the thrashing - either solution is probably fine.